server/routes/api/import_detection/detection.py

```diff
@@ -194,3 +202,148 @@ def detect_columns(col_order: List[Column],
         mapping.update({MappedThing.DATE.value: d_detected})
 
     return json.dumps(mapping, default=vars)
+
+
+@bp.route('/detect_columns', methods=['POST'])
+def detect():
+    """Returns the detected column types and properties.
+
+    Expected required parameters:
+        column_idx_order: List[int]. This is the ordered list of column indices. Indices are expected to be integers.
```

Comment:
I'm wondering if this parameter is necessary because we can just get the order from sorting the keys of column_ids?

Reply:
You are right that it isn't necessary. Got rid of it.